app/assets/javascripts/views/forms/form.js

```diff
@@ -0,0 +1,36 @@
+document.addEventListener("DOMContentLoaded", function () {
```

Comment:
I question whether or not we need a form.js, because technically this javascript will load on all our forms, but the logic will only run for reducer form. 

I think it might be better to have a `<script>` tag with your logic within  the reducer's form partial (views/reducers/_form.html.erb). 

Luckily we also have access to jquery, but I dont mind vanilla js as well :)

Reply:
Yeah, using script makes sense aswell, i was thinking views/forms incase we need other/extra form logic in future but this flies. I've modified the logic to use jquery now and it works just as you shared, vanilla js usuallly comes natural to me by default most times 😊

Reply:
ooh nice!! i literally just free coded that from what little i remember of js + jquery 😆 .